bin/update_cucumber

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env php
+<?php
+
+if (PHP_VERSION_ID < 70400) {
+    echo "Updater requires PHP 7.4";
+    exit(1);
+}
+
+$composerFile = __DIR__ . '/../composer.json';
+$composerConfig = file_get_contents($composerFile);
+foreach (json_decode($composerConfig, true, 512, JSON_THROW_ON_ERROR)['repositories'] as $repository)
+{
+    if ($repository['package']['name'] == 'cucumber/cucumber') {
```

Comment:
I would first do a `if ($repository['type'] !== 'package') { continue; }` so that this script does not break if we add more repositories in the config with other types.